src/core/base/root.ts

```diff
@@ -78,9 +71,8 @@ export class UIPRoot extends ESLBaseElement {
     });
   }
 
-  protected _updateStyles(option: string, prev: string, next: string) {
-    this.classList.remove(`${prev}-${option}`);
-    this.classList.add(`${next}-${option}`);
+  protected _updateStyles(option: string, value: string) {
+    value === null ? this.classList.remove(option) : this.classList.add(option);
```

Comment:
```suggestion
    this.classList.toggle(option, value !== null);
```

Reply:
'force' parameter doesn't work in IE

Reply:
Guys you have no need to care about IE so much, this one basically can be easily covered by polyfill for IE.